yarn.lock

```diff
@@ -7766,7 +7766,12 @@ union-value@^1.0.0:
     is-extendable "^0.1.1"
     set-value "^2.0.1"
 
-universalify@^0.1.0, universalify@^0.2.0:
+universalify@^0.1.0:
```

Comment:
Hmm, any reason why these changes are showing up here now? Is this something that can be skipped and added to `main` after this is merged? Or should it be added before this release goes out?